tests/components/light/test_init.py

```diff
@@ -233,11 +256,18 @@ def test_services(self):
             self.hass, dev1.entity_id,
             profile=prof_name, brightness='bright', rgb_color='yellowish')
 
+        light.set_brightness(
```

Comment:
You should also add a test to check that the service works. You'll have to update the [test light platform](https://github.com/home-assistant/home-assistant/blob/dev/tests/config/custom_components/light/test.py) to allow set_brightness:

```python
from homeassistant.components.light import Light

class Light(Light, MockToggleDevice):
    def __init__(self, name, state):
        """Initialize the mock device."""
        super(MockToggleDevice, self)(name, state)
        self._brightness = 255

    @property
    def brightness(self):
        """Return the brightness of this light between 0..255."""
        return self._brightness
```